charts/redpanda/helpers.go

```diff
@@ -62,8 +62,46 @@ func FullLabels(dot *helmette.Dot) map[string]string {
 		"app.kubernetes.io/component":  Name(dot),
 	}
 
-	// TODO Shouldn't our defaults take precedence over user provided labels?
-	return helmette.Merge(defaults, labels)
+	return helmette.Merge(labels, defaults)
+}
+
+func StatefulSetPodLabelsSelector(dot *helmette.Dot, existingStatefulSetLabelSelector map[string]string) map[string]string {
```

Comment:
nit: Add a short snippet of documentation here?

```
// StatefulSetPodLabelsSelector returns the label selector for the Redpanda StatefulSet.
// If this release is an upgrade, the existing statefulset's label selector will be used as it's an immutable field.


```

Reply:
Done